tests/openmp/issues/CMakeLists.txt

```diff
@@ -3,12 +3,13 @@
 # Distributed under the Boost Software License, Version 1.0. (See accompanying
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fopenmp")
+set(CMAKE_CXX_FLAGS "${CMAKE_C_FLAGS} -fopenmp")
```

Comment:
You probably want for this to read:
```
set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fopenmp") 
```

Reply:
Thanks! I will fix this at next pull request